acronyms.tex

```diff
@@ -0,0 +1,22 @@
+\addtocounter{table}{-1}
+\begin{longtable}{|l|p{0.8\textwidth}|}\hline
+\textbf{Acronym} & \textbf{Description}  \\\hline
+
+ATS & Auxiliary Telescope System \\\hline
+CCD & Charge-Coupled Device \\\hline
+DAQ & Data AcQuisition (system) \\\hline
+DBB & Data BackBone \\\hline
+DM & Data Management \\\hline
+DMCS & DM Control System AAA: Authentication Authorization and Accounting \\\hline
```

Comment:
Something went wrong here because of an error in myacronyms.txt.

Reply:
sorry I missed that - fixed now